.github/workflows/pr.yml

```diff
@@ -32,8 +32,8 @@ jobs:
       - name: Build with Gradle
         run: ./gradlew --no-daemon clean build -x test
         env:
-          ARTIFACTORY_USER: ${{ secrets.ARTIFACTORY_USER }}
-          ARTIFACTORY_APIKEY: ${{ secrets.ARTIFACTORY_APIKEY }}
+          ARTIFACTORY_USER: ${{ secrets.ARTIFACTORY_AUTH_USER }}
```

Comment:
ARTIFACTORY_USER->ARTIFACTORY_AUTH_USER og ARTIFACTORY_APIKEY->ARTIFACTORY_AUTH_TOKEN

Samme gjelder de andre stedene dette er gjort. Se https://github.com/entur/agreements-backend/blob/main/.github/workflows/pr.yml#L35 for et eksempel